.github/workflows/integration.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           go-version: '1.16'
 
       - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+        uses: google-github-actions/setup-gcloud@v0.2.0
```

Comment:
Instead of tag , can you please pin it to a sha? Tags can be moved

Reply:
same as above